src/harness/unittests/tsserverProjectSystem.ts

```diff
@@ -28,7 +28,7 @@ namespace ts.projectSystem {
         })
     };
 
-    export interface PostExecAction {
+    interface PostExecAction {

```

Comment:
PostExecAction is referenced by a protected property in `TestTypingsInstaller`. While this isn't necessarily an error now, it would be if we were to set `"declarations": true` in our tsconfig.json.